ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceBlobSidecarsAvailabilityChecker.java

```diff
@@ -74,31 +113,32 @@ public SafeFuture<BlobSidecarsAndValidationResult> validate(
             return internalValidate(blobSidecars);
```

Comment:
```suggestion
            return internalValidate(blobSidecars, kzgCommitmentsSupplier.get());
```
then remove `BlobSidecarsAndValidationResult internalValidate(final List<BlobSidecar> blobSidecars)` as unused.

Also maybe rename this method to smth like `validateHistorical` to clarify another path of flow. What do you think?

Reply:
I'm more for "feature" naming than "flow" naming... so `validateImmediately` and we can remove the `SafeFuture`